src/middlewared/middlewared/etc_files/local/smb4.conf

```diff
@@ -79,7 +79,17 @@
             smbpasswd by default connects to 127.0.0.1 as an SMB client. For this reason, localhost is added
             to the list of bind ip addresses here.
             """
-            if db['cifs']['bindip']:
+            allowed_interfaces = middleware.call_sync('smb.bindip_choices')
+            validated_bind_ips = []
+            for interface in db['cifs']['bindip']:
```

Comment:
Isnt interface an odd name if its really an IP? or am I misreading what it means?